### PR TITLE
Accept Person's instance or ID when pushing notification

### PR DESCRIPTION
**Pull request by algomaster99**

### CODE REVIEW

File: actions.py [hunk 1]
```diff
@@ -99,6 +104,10 @@ def push_notification(
                 'is True '
             )
         else:
+            persons = [
+                person.id for person in persons if isinstance(person, Person)
```
Comment (pradumangoyal) [inline actions.py]: The else condition will not be handled here, which means that only those ids will be counted for which person instance is passed.

**Review by pradumangoyal — CHANGES_REQUESTED** [review]
Seems something wrong in the implementation of the persons

**Review by pradumangoyal — APPROVED** [review]
Seems fine to me. I think it's better to support instances as well as their ids just like Django ORM handles it.

**Comment by algomaster99** [comment]
@pradumangoyal shall I pull it on the staging server?

**Review by pradumangoyal — APPROVED** [review]

**Pull request merged**